docs/HowTo/Withdrawals.md

```diff
@@ -1,56 +1,69 @@
 # Withdrawals
 
-## Overview
-Validators who stake ether on mainnet, as of the `Bellatrix` upgrade, accrue 2 forms of reward:
-- EL (Execution-layer) rewards payed directly to an eth1address for block inclusion
-- CL (Consensus-layer) rewards for performing actions each epoch, to the CL validator balance.
+Validators staking ether on Mainnet after The Merge, accrue 2 forms of rewards:
 
-Prior to the Capella upgrade, the balances of Consensus-layer validator keys was locked away, and not able to be transferred back to an `eth1Address` (ethereum address).
+- Execution layer rewards payed directly to an Ethereum address.
+- Consensus layer rewards for performing actions each epoch.
 
-The Capella upgrade implements an automated process which allows the funds of validator keys to be deposited to an `eth1Address`.
+Before the Capella upgrade, consensus layer rewards were locked away, and
+unable to be transferred to an Ethereum address. The Capella upgrade implements an automated
+process that allows a validator's funds to be deposited to an Ethereum address.
 
-## Types of withdrawal
-There are two types of withdrawals, and both are automated and require no user interaction once the withdrawal key has been setup as an `eth1Address`.
+!!! note
+
+    Teku provides multiple methods to allow you to
+    [specify an Ethereum address to store your validator's rewards](Prepare-for-The-Merge.md#configure-the-fee-recipient).
```

Comment:
this is separate to withdrawals ... need to be careful because where that one is configurable and changable over time, the bls_to_execution change you do for withdrawals is very permanent.